Tests/GitHubMilestones.tests.ps1

```diff
@@ -0,0 +1,164 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+<#
+.Synopsis
+   Tests for GitHubMilestones.ps1 module
+#>
+
+[String] $root = Split-Path -Parent (Split-Path -Parent $Script:MyInvocation.MyCommand.Path)
```

Comment:
It's odd to force a specific type for a variable.  `Split-Path` already outputs a `[String]` anyways.  Also look into `$PSScriptRoot` because it simplifies this code for you and is automatically available.